assets/js/atomic/blocks/product-elements/rating/block.tsx

```diff
@@ -168,16 +168,18 @@ export const Block = ( props: ProductRatingProps ): JSX.Element | null => {
 		mockedRatings
 	);
 
-	return (
-		<div className={ className } style={ styleProps.style }>
-			<div className="wc-block-components-product-rating__container">
-				{ content }
-				{ reviews && isDescendentOfSingleProductBlock ? (
-					<ReviewsCount reviews={ reviews } />
-				) : null }
+	if ( reviews || shouldDisplayMockedReviewsWhenProductHasNoReviews ) {
```

Comment:
This change introduces a TS error. Please, update the return type at line 137 from:
```
JSX.Element | null
```
to

```
JSX.Element | null
```

Another solution could be add this line after the if:
```
return null
```